Mathlib/Algebra/Order/Incidence.lean

```diff
@@ -0,0 +1,655 @@
+/-
+Copyright (c) 2022 Alex J. Best, Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex J. Best, Yaël Dillies
+-/
+import Mathlib.Algebra.Algebra.Defs
+import Mathlib.Algebra.BigOperators.Intervals
+import Mathlib.Algebra.BigOperators.Ring
+import Mathlib.Algebra.Module.BigOperators
+import Mathlib.Algebra.Module.Pi
+
+/-! -/
+-- fake module docstring just so that I can disable the long line linter without getting linted for
+-- a missing module docstring!
+
+set_option linter.style.longLine false in
+/-!
```

Comment:
@grunweg, you might want to investigate 😇

Reply:
Good question! @damiano You are the expert on this part - with the header linter, what's the best way to silence the linter here? One should certainly be able to do (untested)
```
set_option linter.style header false in
set_option linter.style.longLine false in
stuff